src/workerd/api/tests/actor-stub-test.js

```diff
@@ -13,7 +13,7 @@ export class DurableObjectExample {
     return new Response("OK");
   }
 
-  foo() {
+  async foo() {
     // Server side impl of foo.
     return new String("foo from remote");
```

Comment:
using a boxed string `new String` is really odd. Why not just `return 'foo from remote'`